icu4c/source/test/cintltst/cintltst.h

```diff
@@ -161,6 +161,11 @@ U_CFUNC UBool assertIntEquals(const char* msg, int64_t expected, int64_t actual)
  */
 U_CFUNC UBool assertPtrEquals(const char* msg, const void* expected, const void* actual);
 
+/**
+ * Assert that two doubles are equal, returning TRUE if they do.
```

Comment:
nit: returning TRUE if they _are_.

Reply:
Thanks -- I'll force-push an update in a moment.